themes/clean-detailed.omp.json

```diff
@@ -33,15 +33,14 @@
           "foreground": "#ffffff",
           "leading_diamond": "\ue0b2",
           "style": "diamond",
-          "template": "\uf85a CPU: {{ round .PhysicalPercentUsed .Precision }}% | ",
+          "template": "\uf85a MEM: {{ round .PhysicalPercentUsed .Precision }}% | {{ (div ((sub .PhysicalTotalMemory .PhysicalFreeMemory)|float64) 1000000000.0) }}/{{ (div .PhysicalTotalMemory 1000000000.0) }}GB \uf85a ",
+          "trailing_diamond": "<transparent,#516BEB>\ue0b2</>",
           "type": "sysinfo"
         },
         {
           "background": "#516BEB",
```

Comment:
this entire block should be removed, right?

Reply:
Yes, the entire CPU block

Reply:
This was actually the original `MEM:` block. Sorry for the confusion.